lib/azure/armrest/billing/usage_service.rb

```diff
@@ -0,0 +1,56 @@
+require 'cgi'
```

Comment:
Ah, forgot to remove it. Was thinking I had to encode the dates, but rest-client already does it.